src/main/java/seedu/address/model/person/Person.java

```diff
@@ -102,12 +106,33 @@ public boolean hasMatched(String jobIdentifier) {
      * Returns a string that identify the Person object
      */
     public String getIdentifier() {
-        // TODO: This identifier cannot guarantee uniqueness
-        return name.fullName;
+        return phone.toString();
     }
 
     /**
-     * Returns true if both persons have the same name.
+     * Removes
+     */
+    public void removeMatch() {
+        match = Optional.empty();
+    }
+
+    // Todo: To be closed in PR review, basically we agreed that a person is unique due to contact and email
+    //  and you can see that before this PR, you can add a contact with the same contact or email. Let me know ur thots.
+    ///**
+    // * Returns true if both persons have the same name.
+    // * This defines a weaker notion of equality between two persons.
+    // */
+    //public boolean isSamePerson(Person otherPerson) {
```

Comment:
To be resolved - Do let me know your comments so that I know if I have the green light to proceed or not.

Reply:
Looks like the current implementation defines person equality based on contact or email, which makes sense given our discussion that these fields define uniqueness. 

I agree that we can proceed with this approach and remove the commented-out code that checks only the name. Just make sure to update all related test cases in PersonTest to reflect this logic (checking both contact and email). Once everything is in place, you can delete the TODO comments and proceed with the PR.

Reply:
Understood, I will utilise the one I defined below, which returns if the compared persons have at least a phone number or email in common.